README.md

```diff
@@ -1,2 +1,5 @@
-# Code-review-repo
-To do a own code review for an admission trial project
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Kindly customize this README file to describe what your project is all about.

Reply:
- [ ] Good comment. It is polite and clear 👏🏾